packages/11ty/_plugins/figures/figure-factory.js

```diff
@@ -1,9 +1,10 @@
 const AnnotationFactory = require('./annotation-factory')
 const chalkFactory = require('~lib/chalk')
-const Tiler = require('./iiif/tiler')
-const transform = require('./transform')
 const Manifest = require('./iiif/manifest')
 const ManifestWriter = require('./iiif/manifest/writer')
+const path = require('path')
+const Tiler = require('./iiif/tiler')
+const transform = require('./transform')
```

Comment:
Until we have lint running on pre-commit we might manually sort based on the ESLint [sort-imports](https://eslint.org/docs/latest/rules/sort-imports) rule.